superset-frontend/plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -263,6 +263,7 @@ const config: ControlPanelConfig = {
           },
         ],
         ['adhoc_filters'],
+        ['metrics'],
```

Comment:
check instead what is the other `metrics` panel doing, it must be transforming the props or something

Reply:
Let me check

Reply:
The stranger thing is that it works normally on superset locally.